go/type.go

```diff
@@ -173,9 +174,10 @@ func NewLvalueParser(reader *StringReader) *LvalueParser {
 }
 
 type StringReader struct {
-	i   int
-	pos []pos
-	buf []string
+	i      int
+	pos    []pos
+	buf    []string
+	offset []int
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
field offset is unused (U1000)